internal/osinfo/osinfo_unix.go

```diff
@@ -0,0 +1,70 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+//go:build unix
```

Comment:
I have to add the go:build tag there because of https://github.com/golang/go/issues/20322 so I decided to add build tag to all the files that need it in this package